test/utils/trust.spec.ts

```diff
@@ -85,6 +98,19 @@ describe('Trust Utils method tests', () => {
     ],
   } as BeneficialOwnerOther;
 
+  const mockBoOleCeasedWithTrusteeNoc = {
```

Comment:
I see it's already present in the file, but what's an 'Ole'?

Reply:
other legal entitiy